src/main/resources/lib/samples/sample.jelly

```diff
@@ -44,7 +44,12 @@ THE SOFTWARE.
     <st:adjunct includes="io.jenkins.plugins.designlibrary.styles" />
     <st:adjunct includes="io.jenkins.plugins.designlibrary.sample" />
     <st:adjunct includes="io.jenkins.plugins.designlibrary.preCode" />
-    <st:adjunct includes="io.jenkins.plugins.designlibrary.prism" />
+    <script type="text/javascript" src="${resURL}/plugin/prism-api/js/prism-core.min.js"/>
+    <script type="text/javascript" src="${resURL}/plugin/prism-api/js/prism-markup.min.js"/>
+    <script type="text/javascript" src="${resURL}/plugin/prism-api/js/prism-clike.js"/>
+    <script type="text/javascript" src="${resURL}/plugin/prism-api/js/prism-css.js"/>
+    <script type="text/javascript" src="${resURL}/plugin/prism-api/js/prism-java.min.js"/>
+    <link type="text/css" rel="stylesheet" href="${resURL}/plugin/prism-api/css/${it.themeCssFileName}"/>
```

Comment:
@uhafner this caught me out, it's not defined in the prism.jelly but in the `SourceCodeViewModel` https://github.com/jenkinsci/prism-api-plugin/blob/main/src/main/resources/io/jenkins/plugins/prism/SourceCodeViewModel/index.jelly#L10

seemed a bit strange that the plugin didn't load it, some sort of invoke static could probably be used to avoid having to define the getter?

Reply:
Yes, it would make sense to autoload the theme somehow in the main jelly file. Since the warnings plugin is the only component so far that uses the plugin it was not yet necessary up to now. I'm not sure how to store that information on the JS side though. I think when we make the theming customizable via the theme manager plugin we should change that...

Reply:
Fixed by integrating it in here:
https://github.com/timja/prism-api-plugin/blob/1064ebc55f37974413526b1da3913f5407f35232/src/main/webapp/js/theme-loader.js#L24-L27